calico/felix/splitter.py

```diff
@@ -107,7 +142,7 @@ def on_endpoint_update(self, endpoint_id, endpoint):
         for mgr in self.ep_upd_mgrs:
             mgr.on_endpoint_update(endpoint_id, endpoint, async=True)
 
-    def on_ipam_pool_update(self, pool_id, pool):
+    def on_ipam_pool_updated(self, pool_id, pool):
```

Comment:
It was a side-effect of the work I did to clean up the splitter UTs, forced by adding one-too-many methods.